includes/REST_API/Stories_Users_Controller.php

```diff
@@ -111,34 +123,15 @@ public function get_items( $request ) {
 	 * @return array Filtered args.
 	 */
 	public function filter_user_query( $prepared_args ) {
-		global $wpdb;
+		$registered = $this->get_collection_params();
 
-		$blog_id = get_current_blog_id();
-		if ( isset( $prepared_args['blog_id'] ) ) {
-			$blog_id = absint( $prepared_args['blog_id'] );
-		}
+		// Capability queries were added in 5.9, and the 'who' param was deprecated.
+		if ( isset( $prepared_args['who'], $registered['capabilities'] ) && 'authors' === $prepared_args['who'] ) {
+			$capabilities   = $prepared_args['capabilities'] ?? [];
+			$capabilities[] = $this->story_post_type->get_cap_name( 'edit_posts' );
+
+			$prepared_args['capabilities'] = $capabilities;
```

Comment:
Should this change be made in the core user endpoint

Reply:
This would prevent devs from seeing the deprecation notice and updating their code to use the new capabilities query.

Reply:
Also, why the thumbs down? 🤔 

This is more forward compatible than just doing a user level meta query despite that practice technically being deprecated.